bin/jupyter-kernel.raku

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl6
+#!/usr/bin/env raku
```

Comment:
Is a `raku` executable available for os/x without building from source?

Reply:
Not on my Ubuntu 19.10